app/views/core/CreateAccountModal/BasicInfoView.coffee

```diff
@@ -220,7 +220,7 @@ module.exports = class BasicInfoView extends CocoView
       # update User
       emails = _.assign({}, me.get('emails'))
       emails.generalNews ?= {}
-      emails.generalNews.enabled = @$('#subscribe-input').is(':checked') and not _.isEmpty(@state.get('checkEmailValue'))
+      emails.generalNews.enabled = not _.isEmpty(@state.get('checkEmailValue'))
```

Comment:
Is it the new behavior that all users with emails are by default opted into emails, and then only opted out if they uncheck this checkbox and click "Finish"?

This might be OK, but it has the unfortunate effect that if you uncheck the checkbox and then close the modal some other way (escape, click the X) it will not actually unsubscribe them.